srcCpp/RTIDDSImpl.h

```diff
@@ -210,12 +217,13 @@ class RTIDDSImpl : public IMessaging
     RTIOsapiSemaphore *_pongSemaphore;
     RTIDDSLoggerDevice _loggerDevice;
 
+    std::map<std::string, std::string> _qoSProfileNameMap;
+
   public:
```

Comment:
shouldn't public and private be at the same level?

Reply:
Yes, Done